ts/src/renderer/phaser/classes/PhaserProjectile.ts

```diff
@@ -45,10 +44,13 @@ class PhaserProjectile extends Phaser.GameObjects.Container {
 			});
 
 		this.destroyListener = projectile.on('destroy', () => {
-			projectile.off('play-animation', this.playAnimationListener);
+			projectile.off('transform', this.playAnimationListener);
 			this.transformListener = null;
+			projectile.off('scale', this.playAnimationListener);
 			this.scaleListener = null;
+			projectile.off('play-animation', this.playAnimationListener);
 			this.playAnimationListener = null;
+			projectile.off('destroy', this.playAnimationListener);
```

Comment:
these all say `this.playAnimationListener` :P

Reply:
:/